### PR TITLE
Create Initial Data Schema

### PR DESCRIPTION
**Pull request by bat-kryptonyte**
## PR Create Initial Data Schema

Issue Number(s): #11

What does this PR change and why?
Added new schema fields

### Related PRs 
None

### How to Test

yarn dev

MongoDB
A running instance of MongoDB is required this project.

Locally (Docker - RECOMMENDED)
[Download Docker Desktop](https://www.docker.com/products/docker-desktop)
Run docker run --name mongodb -d -p 27017:27017 mongo in your terminal
Open Docker Desktop and confirm that your MongoDB image is running. It should exist on port 27017, and can be accessed.
Locally (Non-Docker)
[Download MongoDB Community Server](https://www.mongodb.com/download-center/community)
Go through the installation instructions.
Leave the port at default 27017

### CODE REVIEW

File: server/mongodb/models/InitSchema.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+const project = new mongoose.Schema({
+    name: {
+        type: String,
+        required: true,
+    },
+    semesterCreated: {
+        type: String, 
+        enum: ['spring','summer','fall'],
+        required: true,
+    },
```
Comment (kphan32) [inline server/mongodb/models/InitSchema.js:28]: The semester declaration should be defined at the top and re-used

File: server/mongodb/models/InitSchema.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+const userSchema = mongoose.model("userSchema", user);
+const projectSchema = mongoose.model("projectSchema", project);
+const tenureSchema = mongoose.model("tenureSchema", tenure);
+
+module.exports = {
+    userSchema: userSchema,
+    projectSchema: projectSchema,
+    tenureSchema: tenureSchema
+}
```
Comment (kphan32) [inline server/mongodb/models/InitSchema.js:77]: 🤔 prettier?

**Review by kphan32 — CHANGES_REQUESTED** [review]

**Comment by kphan32** [comment]
oh-- also don't forget the pull in the fields in the `User` schema defined by `next-auth`.

**Comment by kphan32** [comment]
also also.. just updated the original ticket to remove the projects schema. realized that it probably isn't needed

**Comment by therealsharath** [comment]
@bat-kryptonyte could you please re-request a review and add me as a reviewer? 🤔

**Pull request closed without merging**